app/build.gradle

```diff
@@ -34,6 +39,8 @@ dependencies {
     implementation 'com.google.android.material:material:1.3.0'
     implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
     testImplementation 'junit:junit:4.+'
```

Comment:
This is generated by Android Studio, but try to replace it by a specific version or it can be problematic later on.

Reply:
Replaced!
![image](https://user-images.githubusercontent.com/85253636/122766774-462a0700-d278-11eb-8500-a53b81388678.png)